scheds/rust/scx_layered/src/main.rs

```diff
@@ -2699,6 +2930,7 @@ impl<'a> Scheduler<'a> {
         )?;
         self.refresh_cpumasks()?;
         self.refresh_idle_qos()?;
+        self.gpu_task_handler.process();
```

Comment:
We prioritize userspace scheduler thread quite a bit and listing all PIDs in the system can take a while. Given that this isn't latency sensitive, it likely would be a good idea to spawn this into a separate thread which is kicked off from `.init()` and update `is_scheduler_task()` to exclude the thread. Given that what this does doesn't have much to do with the scheduling loop, this can also be moved into its own process. ie. Make it a subcommand of `scx_layered`.